recipe/handle_visual_studio_forceinline.patch

```diff
@@ -0,0 +1,24 @@
+From ceca435d0045e1042f4045a0c43e98d7d7e69df8 Mon Sep 17 00:00:00 2001
+From: Serhii Kupriienko
```

Comment:
This patch is not from you. Please give proper attributions. You can use https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/474.patch to get the full patch.

Reply:
Thanks. Will do